DRX/Fringing/fringeSDF.py

```diff
@@ -128,7 +134,14 @@ def main(args):
         print("Setting gains for %i good inputs, %i bad inputs" % (len(antennas)-len(bad), len(bad)))
         print("-> Using gain setting of %.4f for the beam" % bgain)
         
-        gains = [[twoByteSwap(gain_to_dpg(g)) for g in baseBeamGain] for i in xrange(260)] # initialize gain list
+        gains = [[twoByteSwap(gain_to_dpg(g)) for g in baseBeamGain] for i in xrange(int(len(antennas)/2))] # initialize gain list 
+        #If achromatic beamforming is requested, set the gains for it.
+        if args.custom:
+            #These weird gain values will signal to ADP to use the predetermined
+            #achromatic beam gains located in /home/adp/.
+            gains[0] = [8191,16383,32767,65535]
+            gains[1] = [1,1,1,1]
+
```

Comment:
I'm not sure about adding an option for custom beamforming in this script.  It's not very general purpose at this stage and it's completely undocumented.